inspirehep/modules/workflows/tasks/submission.py

```diff
@@ -119,7 +122,8 @@ def reply_ticket(template=None,
     @with_debug_logging
     @wraps(reply_ticket)
     def _reply_ticket(obj, eng):
-        ticket_id = obj.extra_data.get("ticket_id", "")
+        ticket_id_key = "ticket_id"
+        ticket_id = obj.extra_data.get(ticket_id_key, "")
```

Comment:
couldn't you return an empty `dict`in this case, as this function is not writing anything to `extra_data`?